Makefile

```diff
@@ -1,4 +1,6 @@
+.PHONY: install
+
 install:
 	./scripts/github-install.sh
 
-include scripts/Makefile
+include ./scripts/Makefile
```

Comment:
It's for consistency with the line above. I ran on Ubuntu before this change without issues.